content/references/protocol-reference/transactions/transaction-common-fields.md

```diff
@@ -12,7 +12,7 @@ Every transaction has the same set of common fields, plus additional fields base
 | Field                | JSON Type        | [Internal Type][] | Description      |
 |:---------------------|:-----------------|:------------------|:-----------------|
 | `Account`            | String           | AccountID         | _(Required)_ The unique address of the [account](accounts.html) that initiated the transaction. |
-| `TransactionType`    | String           | UInt16            | _(Required)_ The type of transaction. Valid types include: `Payment`, `OfferCreate`, `OfferCancel`, `TrustSet`, `AccountSet`, `AccountDelete`, `SetRegularKey`, `SignerListSet`, `EscrowCreate`, `EscrowFinish`, `EscrowCancel`, `PaymentChannelCreate`, `PaymentChannelFund`, `PaymentChannelClaim`, and `DepositPreauth`. |
+| `TransactionType`    | String           | UInt16            | _(Required)_ The type of transaction. Valid [types](transaction-types.html) include: `Payment`, `OfferCreate`, `OfferCancel`, `TrustSet`, `AccountSet`, `AccountDelete`, `SetRegularKey`, `SignerListSet`, `EscrowCreate`, `EscrowFinish`, `EscrowCancel`, `PaymentChannelCreate`, `PaymentChannelFund`, `PaymentChannelClaim`, `DepositPreauth`, ... |
```

Comment:
I agree with the premise here, but if anything I don't think you take it far enough. If you're going to abbreviate it, let's trim it to three example types and leave the rest to if you click through.

```suggestion
| `TransactionType`    | String           | UInt16            | _(Required)_ The type of transaction. Valid [transaction types](transaction-types.html) include: `Payment`, `OfferCreate`, `TrustSet`, and many others. |
```